tests/build.proj

```diff
@@ -31,6 +31,7 @@
     <RestoreProjects Include="$(MSBuildThisFileDirectory)src\JIT\config\benchmark+serialize\benchmark+serialize.csproj" />
     <RestoreProjects Include="$(MSBuildThisFileDirectory)src\JIT\config\benchmark\benchmark.csproj" />
     <RestoreProjects Include="$(MSBuildThisFileDirectory)src\performance\performance.csproj" />
+    <RestoreProjects Include="$(MSBuildThisFileDirectory)src\readytorun\r2rdump\R2RDumpTest.csproj" />
```

Comment:
This looks unnecessary since R2RDumpTest.csproj doesn't have any PackageReferences.

Reply:
When I didn't have this, the project.assets.json of the test programs such as files/MultipleRuntimeFunctions doesn't get generated